util/es_template.go

```diff
@@ -0,0 +1,19 @@
+package util
+
+import (
+	"context"
+)
+
+// SetDefaultIndexTemplate to set default template for indexes
+func SetDefaultIndexTemplate() error {
+	response, err := GetClient7().IndexTemplateExists("default_temp").
```

Comment:
@lakhansamani Does it work for `ES6`?
Did you verify it?

Reply:
Yes verified with ES6 it works 👍